Sources/Sentry/SentryDevice.h

```diff
@@ -0,0 +1,48 @@
+#import <Foundation/Foundation.h>
```

Comment:
Can you remind me what happens if I don't put it in either location? I'm guessing there's some downstream effect with package managers? Nothing failed in CI due to this. I don't think this should be public, FWIW.

Reply:
If it's public and you don't put it in `Sources/Sentry/Public` CI will fail because of package managers yes. Putting nonpublic headers in `Sources/Sentry/include` is just a pattern we use to follow. I'm also open to discussion on changing this.

Reply:
I think for now i'll just stick to putting it in `include`. Not really interested in reworking things now since the rest is a flat directory of files 😅